search.xml

```diff
@@ -1,6 +1,9 @@
+---
+---

```

Comment:
Not sure where these cpme from.

Reply:
Oh, I see. To be able to use `{{ site.url }}`, this is now a Jekyll template file.
All good.